libs/application-generic/src/services/feature-flags/launch-darkly.service.ts

```diff
@@ -61,21 +61,21 @@ export class LaunchDarklyFeatureFlagsService implements IFeatureFlagsService {
       kind: 'multi',
     };
 
-    if (environment) {
+    if (environment?._id) {
```

Comment:
i was not able to find the main source of the issue, but we were sending kind of`undefined` which is not allowed on LD side.